experimental/dds/tree/docs/Breaking-Change-Migration.md

```diff
@@ -0,0 +1,52 @@
+# Breaking Change Migration
```

Comment:
Are there any docs on the breaking API changes for teams using the current tree from FF? I suspect having the API changes documented will make migrations easier for partners.

Reply:
Normally, yes, I agree. However, given timing/deadlines, we took a different approach: there's a small set of consumers (this is marked experimental, after all) and I've worked with each of them to integrate this code already. We could probably do a fast-follow with some documentation on migrating from 0.0.2 to 0.1.1 APIs, but in general I don't think anyone else is using it yet.

Reply:
I'm going to resolve this to unblock this before the branch cut, but we can follow up about supporting it :)

Reply:
I added a note to the top of the PR mentioning in theory partner teams should be able to scrub the history to find whatever breaking change is affecting them (and if not, to reach out to me as viewing the history on the Whiteboard end might make this clearer). But yeah, in practice we think all teams have already been notified and in many cases have working prototypes against these changes.